packages/esm-patient-registration-app/src/patient-registration/patient-registration.component.tsx

```diff
@@ -65,16 +65,16 @@ export const PatientRegistration: React.FC<PatientRegistrationProps> = ({ savePa
       .filter((s) => s);
   }, [config.sections, config.sectionDefinitions]);
 
-  useEffect(() => {
-    if (addressTemplate) {
-      const addressTemplateXml = addressTemplate?.results[0].value;
-      if (!addressTemplateXml) {
-        return;
-      }
-      const { addressValidationSchema } = parseAddressTemplateXml(addressTemplateXml);
-      setValidationSchema((validationSchema) => validationSchema.concat(addressValidationSchema));
-    }
-  }, [inEditMode, addressTemplate, initialAddressFieldValues]);
+  // useEffect(() => {
```

Comment:
We can delete this right?

Reply:
Yes, I missed it.

Reply:
Hi @ibacher !
This brings to another question. 
Do we need the regex validation from the address template to work on the frontend?
Thanks!

Reply:
@vasharma05 Probably eventually, but it's not the highest priority.